layouts/index.html

```diff
@@ -105,11 +100,11 @@ <h2><i class="fa-light fa-chart-network"></i> Open Source Ecosystem </h2>
 
             <div class="row has-background-rapids-image ">
                 <div class="col-md-12 py-5">
-                    <h2 class="text-white"><i class="fa-light fa-gauge-circle-bolt"></i> NEW: Pandas Accelerator Mode
+                    <h2 class="text-white"><i class="fa-light fa-gauge-circle-bolt"></i> Pandas Accelerator Mode
                         for cuDF </h2>
-                    <p class="text-white">Use cuDF's new pandas Accelerator Mode to speed up pandas workflows with zero
+                    <p class="text-white">Use cuDF's pandas Accelerator Mode to speed up pandas workflows with zero
```

Comment:
Avoid the possessive here.
```suggestion
                    <p class="text-white">Use cuDF pandas Accelerator Mode to speed up pandas workflows with zero
```